Annotato/Annotato/SceneDelegate.swift

```diff
@@ -22,15 +28,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         }
 
         let documentListViewController = DocumentListViewController.instantiateFullScreenFromStoryboard(.document)
-
-        let authViewController = AuthViewController.instantiateFullScreenFromStoryboard(.main)
-
         documentListViewController?.webSocketManager = webSocketManager
-        authViewController?.webSocketManager = webSocketManager
 
         window?.rootViewController = AuthViewModel().currentUser != nil
             ? documentListViewController
```

Comment:
This case is already handled by the WebSocketManager. If the user goes straight into the `documentListViewController` then it would trigger the network change publisher.

Reply:
@ganhongyao if there are any problems with this, i will do it in a another PR